models/database.php

```diff
@@ -6,8 +6,8 @@
 class Database {
   private $host = "localhost";
   private $dbname = "foroweb";
-  private $dbuser = "isma";
-  private $dbpass = '123';
+  private $dbuser = "root";
+  private $dbpass = '';
```

Comment:
it is not a good  practices leave your user and password you use in your environment